src/lines-component.coffee

```diff
@@ -343,59 +114,18 @@ class LinesComponent
 
   measureCharactersInNewLines: ->
     @presenter.batchCharacterMeasurement =>
-      for id, lineState of @oldState.lines
-        unless @measuredLines.has(id)
-          lineNode = @lineNodesByLineId[id]
-          @measureCharactersInLine(id, lineState, lineNode)
-      return
```

Comment:
No worries about that. I am actually sorry [this diff got so ugly](https://github.com/atom/atom/pull/6733/files#r30785003) :pray: